### PR TITLE
please add new project matrix-commander-rs to your list

### PR DESCRIPTION
**Pull request by 8go**
Preview: https://pr1496--matrix-org-previews.netlify.app

### CODE REVIEW

**Comment by 8go** [comment]
@thibaultamartin 

Sorry to bother you, but I need help. I do not understand why the Gatsby builds are failing? What am I doing wrong? Where is the problem? Please kindly take a minute and assist. I am stuck. Thank you.

File: gatsby/content/projects/clients/matrix-commander-rs.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+# matrix-commander-rs
+
+simple but convenient CLI-based Matrix client app for sending and receiving
+
+# Help create this Rust program!
+
+Currently you can log in, verify your account, send unencrypted messages, send 
+encypted messages, send encrypted files, list devices, and log out.
```
Comment (clokep) [inline gatsby/content/projects/clients/matrix-commander-rs.mdx:28]:
```suggestion
encrypted messages, send encrypted files, list devices, and log out. 
```

**Comment by 8go** [comment]
OK, it went through. Ignore the previous call for help. Thank you!

When will it be "online"? Roughly?

**Comment by 8go** [comment]
An old template had like a list of features, you don't need those anymore in the .mdx file? Correct?

**Comment by 8go** [comment]
Is there anything I need to do on my side to move this forward? thank you for your help.

**Comment by thibaultamartin** [comment]
Thanks for patience, this will be deployed shortly on matrix.org :)

**Pull request merged**